pkgs/applications/emulators/bsnes/ares/default.nix

```
@@ -15,6 +16,8 @@
 , libpulseaudio
 , openal
 , udev
+, Cocoa
+, OpenAL
```

Given the issue #204303, 
```suggestion
, darwin
```

---

Done.